modules/dreamview/frontend/src/renderer/obstacles.js

```diff
@@ -350,8 +412,14 @@ export default class PerceptionObstacles {
     return arrowMesh;
   }
 
-  getFace(index, scene, solid = true) {
-    const extrusionFaces = solid ? this.extrusionSolidFaces : this.extrusionDashedFaces;
+  getFace(index, scene, solid = true, v2x = false) {
```

Comment:
Better to create an enum for faceType, which contains DASHED_LINE, SOLID_LINE, and SOLID_FACE, then passed in the the type to this getFace function.

Same for getCube().